### PR TITLE
Octopus Deploy's Log4J Response

### PR DESCRIPTION
**Pull request by wizedkyle**
# About

This post is about our response to the Log4J vulnerability. 

# Release

N/A

# Blog image idea

Added the usual security image banner.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

## How to review this PR

Can you please review the content and provide any suggestions for changing.

## Publication date

Can we publish this ASAP please.

### CODE REVIEW

File: blog/2021-12/octopus-deploy-log4j-response/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+On Thursday, December 9, 2021, Octopus Deploy was made aware of a vulnerability in the Log4J logging utility ([CVE-2021-44228](https://cve.mitre.org/cgi-bin/cvename.cgi?name=2021-44228)). We immediately enacted our Security Incident Response Plan to investigate our use of this utility and its impact on our products and infrastructure.
+
+Fortunately, Octopus Cloud, Octopus Server, and Octopus Tentacle are not affected. However, there are some related products that are impacted and require updates.
+
+In this post, we share the results of our investigation and our recommended next steps for customers.
+
+## JetBrains TeamCity plugin
+
+Version 6.1.7 of the JetBrains TeamCity plugin has been updated to use version 2.15.0 of Log4J. However, all versions of the plugin before 6.1.7 are using version 2.14.1 of Log4J and could be vulnerable to remote code execution.
```
Comment (kathrynmarks) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:24]: I didn't pick this up when editing the email copy 🤦 , but we updated the plugin _to_ v6.1.7. If we updated v6.1.7 then we'd have the next version number. This is a techy nitpicking detail that some people in our audience may notice.

```suggestion
Version 6.1.7 of the JetBrains TeamCity plugin uses version 2.15.0 of Log4J. However, all versions of the plugin before 6.1.7 are using version 2.14.1 of Log4J and could be vulnerable to remote code execution.
```

File: blog/2021-12/octopus-deploy-log4j-response/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  - Trust and Security
+---
+
+On Thursday, December 9, 2021, Octopus Deploy was made aware of a vulnerability in the Log4J logging utility ([CVE-2021-44228](https://cve.mitre.org/cgi-bin/cvename.cgi?name=2021-44228)). We immediately enacted our Security Incident Response Plan to investigate our use of this utility and its impact on our products and infrastructure.
+
+Fortunately, Octopus Cloud, Octopus Server, and Octopus Tentacle are not affected. However, there are some related products that are impacted and require updates.
+
+In this post, we share the results of our investigation and our recommended next steps for customers.
+
+## JetBrains TeamCity plugin
```
Comment (kathrynmarks) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:22]: Suggestion for specificity.

```suggestion
## JetBrains TeamCity plugin for Octopus Deploy
```

File: blog/2021-12/octopus-deploy-log4j-response/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Version 6.1.7 of the JetBrains TeamCity plugin has been updated to use version 2.15.0 of Log4J. However, all versions of the plugin before 6.1.7 are using version 2.14.1 of Log4J and could be vulnerable to remote code execution.
+
+### What you need to do
+
+Version 6.1.7 of the Octopus Deploy plugin for TeamCity has been published to the marketplace. If you're a TeamCity user, you should be notified of this update through the Web UI and you're strongly encouraged to update.
```
Comment (kathrynmarks) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:28]: Another option to omit the passive voice.

```suggestion
Version 6.1.7 of the Octopus Deploy plugin for TeamCity is available from the marketplace. If you're a TeamCity user, you should be notified of this update through the Web UI and you're strongly encouraged to update.
```

Comment (kathrynmarks) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:28]: I've been trying to improve the second sentence, but I'm struggling. 

It's unclear whether the update notification is through the Octopus UI or a TeamCity UI. My brain also wants to see something at the end of the sentence. Perhaps `the plugin`.

File: blog/2021-12/octopus-deploy-log4j-response/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Follow our [security advisory](https://advisories.octopus.com/adv/2021-12---Octopus-Deploy-TeamCity-Plugin-log4j2-dependency.2306410241.html) for updates.
+
+## Octopus Java SDK
+
+Version 0.0.3 of the Java SDK has been updated to use version 2.15.0 of Log4J. However, all versions of the Java SDK before 0.0.3 are using version 2.14.1 of Log4J and could be vulnerable to remote code execution.
```
Comment (kathrynmarks) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:34]: (as above)

```suggestion
Version 0.0.3 of the Java SDK uses version 2.15.0 of Log4J. However, all versions of the Java SDK before 0.0.3 are using version 2.14.1 of Log4J and could be vulnerable to remote code execution.
```

**Review by kathrynmarks — COMMENTED** [review]
Looks great. I caught something I missed in the email copy and made a few other suggestions.

File: blog/2021-12/octopus-deploy-log4j-response/index.md (hunk 3 above)
Comment (kathrynmarks) [reply blog/2021-12/octopus-deploy-log4j-response/index.md]: Alternatively, you could end the sentence with something like `you're strongly encouraged to accept the update`.

Comment (kathrynmarks) [reply blog/2021-12/octopus-deploy-log4j-response/index.md]: Even better, less passive, `we strongly encourage you to accept the update`.

Comment (wizedkyle) [reply blog/2021-12/octopus-deploy-log4j-response/index.md]: Can we go with `we strongly encourage you to install the update` ?

File: blog/2021-12/octopus-deploy-log4j-response/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+In this post, we share the results of our investigation and our recommended next steps for customers.
+
+## JetBrains TeamCity plugin for Octopus Deploy
+
+Version 6.1.7 of the JetBrains TeamCity plugin uses version 2.15.0 of Log4j. However, all versions of the plugin before 6.1.7 are using version 2.14.1 of Log4j and could be vulnerable to remote code execution.
+
+### What you need to do
+
+Version 6.1.7 of the Octopus Deploy plugin for TeamCity is available from the marketplace. If you're a TeamCity user, you should be notified of this update through the Web UI and you're strongly encouraged to update the plugin.
```
Comment (TeganAli) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:28]:
```suggestion
Version 6.1.7 of the Octopus Deploy plugin for TeamCity is available from the marketplace. If you're a TeamCity user, you should be notified of this update through the Web UI and we strongly encourage you to accept the update.
```

File: blog/2021-12/octopus-deploy-log4j-response/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+In this post, we share the results of our investigation and our recommended next steps for customers.
+
+## JetBrains TeamCity plugin for Octopus Deploy
+
+Version 6.1.7 of the JetBrains TeamCity plugin uses version 2.15.0 of Log4j. However, all versions of the plugin before 6.1.7 are using version 2.14.1 of Log4j and could be vulnerable to remote code execution.
+
+### What you need to do
+
+Version 6.1.7 of the Octopus Deploy plugin for TeamCity is available from the marketplace. If you're a TeamCity user, you should be notified of this update through the Web UI and we strongly encourage you to accept the update.
```
Comment (TeganAli) [inline blog/2021-12/octopus-deploy-log4j-response/index.md:28]:
```suggestion
Version 6.1.7 of the Octopus Deploy plugin for TeamCity is available from the marketplace. If you're a TeamCity user, you should be notified of this update through the Web UI and we strongly encourage you to install the update.
```

File: blog/2021-12/octopus-deploy-log4j-response/index.md (hunk 3 above)
Comment (TeganAli) [reply blog/2021-12/octopus-deploy-log4j-response/index.md]: Thanks @kathrynmarks and @wizedkyle - I've updated to `we strongly encourage you to install the update`

**Pull request merged**